app/models/board.rb

```diff
@@ -0,0 +1,54 @@
+class Board < ApplicationRecord
+  include ActsAsLinkedList
+  list_item_name :list
+
+  # -- Relationships --------------------------------------------------------
+  belongs_to :node
+  has_many :activities, as: :trackable
+  # NOTE: Rails >= 5.1 raises error when a `has_many :y, through: x`
+  # relationship is defined before the corresponding `has_many :x` relationship.
+  # (https://github.com/rails/rails/pull/27485).
+  # So we define `lists` before `cards`
+  has_many :lists, dependent: :destroy
+  has_many :cards, through: :lists
+
+  def project
```

Comment:
these 2 methods are the new thing in CE. And there is no `belongs_to :project` in CE.